src/citrine/informatics/predictors/node.py

```diff
@@ -0,0 +1,61 @@
+import warnings
+from typing import Type, Optional, List
+
+from citrine._serialization import properties
+from citrine._serialization.polymorphic_serializable import PolymorphicSerializable
+from citrine.informatics.data_sources import DataSource
+
+
+def _check_deprecated_training_data(training_data: Optional[List[DataSource]]) -> None:
+    if training_data is not None:
+        warnings.warn(
+            f"The field `training_data` on single predictor nodes is deprecated "
+            "and will be removed in version 3.0.0. Include training data for all "
+            "sub-predictors on the parent GraphPredictor. Existing training data "
+            "on this predictor will be moved to the parent GraphPredictor upon registration.",
+            DeprecationWarning
+        )
+
+
+class PredictorNode(PolymorphicSerializable["PredictorNode"]):
```

Comment:
It is not possible for the node types to acquire meta-data/version information because they cannot be registered standalone. As such, I have removed those mix-in classes from the inheritance chain which would indicate a break in the fields present on the types. 

They can be added back for the sake of backwards compat if we deem necessary.

Reply:
These are all read-only fields, right? So the potential break is that someone could have a script in which they request, say the `updated_by` field of a predictor node, and that field is no longer available. Is that correct?

If it's impossible to register non-graph predictor nodes, then none of those fields are meaningful anyway. We'd want to pull the associated value from the graph predictor. Not that that's easy to do...

Reply:
>If it's impossible to register non-graph predictor nodes, then none of those fields are meaningful anyway

These fields are only populated when you register a predictor. They are generated by the platform upon DB entry. Given that all predictor nodes are contained within the Graph, I do not think it is possible for these fields to be populated on the node classes.

So for example, lets say a user creates a local `AutoMLPredictor` and tries to access the `updated_by` field. Before these changes, that would result in a value of `None` being returned. After these changes, it would become `'AutoMLPredictor' object has no attribute 'updated_by'`

I suppose one option is to just leave the inheritance of those types and the fields can never be populated since the platform will never fill them in.

Reply:
Let's just leave the inheritance. It maintains strict backwards compatibility and is very easy to remove when we go to version 3

Reply:
The inheritance fucks with the serialization paths for the data, so this is quite tedious to figure out.